_playwright-tests/UI/PopularRepo.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { describe } from 'node:test';
+import { test, type Page } from '@playwright/test';
+import { navigateToRepositories } from './helpers/navHelpers';
+import { deleteAllRepos } from './helpers/deleteRepositories';
+import { closePopupsIfExist } from './helpers/helpers';
+
+describe('Popular Repositories', () => {
+  test('Clean - Delete any current repos that exist', async ({ page }) => {
+    await deleteAllRepos(page);
+  });
+
+  test('Navigate to Popular Repos tab, assert the Add button is displayed', async ({ page }) => {
+    await navigateToRepositories(page);
+    await closePopupsIfExist(page);
+
+    const AddMenuButton = page.locator('[data-ouia-component-id="add_checked_repos-with-snap"]');
+    await page.getByRole('link', { name: 'Popular repositories' }).click();
+
+
+    // Wait for the Add button to become enabled (up to 10 seconds)
+    await AddMenuButton.first().isEnabled({ timeout: 10000 });
+  });
+
+  test('Add popular repos', async ({ page }) => {
+    await navigateToRepositories(page);
+    await closePopupsIfExist(page);
+
+    const AddMenuButton = page.locator('[data-ouia-component-id="add_popular_repo_toggle"]');
```

Comment:
This does not seem to be used? :thinking:

Reply:
will remove